File: FirstMVVM VB/FirstMVVM/ViewModels/FirstViewModel.vb
Imports Caliburn.Micro

Public Class FirstViewModel
    Inherits Screen
    Private _FirtName As String
    Public Property FirtName() As String
        Get
            Return _FirtName
        End Get
        Set(ByVal value As String)
            _FirtName = value
            NotifyOfPropertyChange(Function() FirtName)
            NotifyOfPropertyChange(Function() FullName)
        End Set
    End Property

    Private _SecondName As String
    Public Property SecondName() As String
        Get
            Return _SecondName
        End Get
        Set(ByVal value As String)
            _SecondName = value
            NotifyOfPropertyChange(Function() SecondName)
            NotifyOfPropertyChange(Function() FullName)
        End Set
    End Property

    Private _FullName As String
    Public Property FullName() As String
        Get
            Return FirtName + SecondName
        End Get
        Set()

        End Set
    End Property
End Class

File: FirstMVVM C#/FirstMVVM/ViewModels/FirstViewModel.cs
using Caliburn.Micro;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FirstMVVM.ViewModels
{
   public  class FirstViewModel:Screen
    {
        private string _FirtName = "向量";

        public string FirtName
        {
            get { return _FirtName; }
            set
            {
                _FirtName = value;
                NotifyOfPropertyChange(() => FirtName);
                NotifyOfPropertyChange(() => FullName);
            }
        }

        private string _SecondName;

        public string SecondName
        {
            get { return _SecondName; }
            set
            {
                _SecondName = value;
                NotifyOfPropertyChange(() => SecondName);
                NotifyOfPropertyChange(() => FullName);
            }
        }

       
        public string FullName
        {
            get { return FirtName  + SecondName; }
             
        }

    }
}
